### DISCUSSION TITLE
Github Account primary mail unlinked

### DISCUSSION THREAD

**Discussion by shuklaRishabh46** [community/community#124559] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I am a user of the GitHub account, which has been my primary account for years. In 2021, I merged this account with my organization email, making it the primary email with enabled 2FA authentication.

I left the organization in December 2022, and since then, I have been locked out of my account because the primary email was deactivated. Despite multiple attempts using various methods found online, I have been unable to regain access.

During this process, my original email appears to have been unlinked from my GitHub account, leaving me with no way to access it, as it now shows no account at all.

I tried mailing support@github.com but it says, "Message bounced due to organizational settings."

Is it possible at all to access my account now?

**Comment by dcaayushd** [comment]
Regaining access to your GitHub account after losing access to the primary email associated with it can be a challenging process, but there are a few steps you can try:

1. **Contact GitHub Support**: Although the email you sent to support@github.com bounced, you can try contacting GitHub Support through other channels. Visit the GitHub Support Portal (https://support.github.com/) and click on "Submit a request" or "Contact GitHub Support" to open a new support ticket. Provide as much detail as possible about your situation, including your GitHub username, the email addresses previously associated with your account, and any other relevant information that could help them identify your account.

2. **Provide Proof of Ownership**: GitHub may ask you to provide proof of ownership for your account. This could include details about your account activity, such as the repositories you've contributed to, issues or pull requests you've created, or other information that only the legitimate account owner would know.

3. **Recover Access to the Original Email**: If possible, try to regain access to the original email address that was associated with your GitHub account before you merged it with your organization's email. GitHub may be able to help you relink that original email to your account if you can access it again.

4. **Check for Account Recovery Options**: When you try to log in to your GitHub account, there may be options to recover your account or reset your password. These options may be available if you still have access to any of the email addresses or recovery codes previously associated with your account.


It's important to note that GitHub takes account security and ownership very seriously, and they may require substantial proof of ownership before granting access to an account, especially if there are concerns about unauthorized access attempts.

While regaining access can be challenging in your situation, being persistent and providing as much information as possible to GitHub Support may increase your chances of successfully recovering your account.

**Reply by shuklaRishabh46** [reply]
Thanks for replying. I have tried raising the ticket but it gets stuck at virtual assistant always.

I do have access to my primary mail, but that seems to be unlinked with my email. When I try forget password it ask for my email, upon entering says no account on this email. I am ready to prove my ownership for the same. That account was linked to the email which is mine, Photo was also there which is mine, Mentioned linkdin URL is mine as well(added when I still had accessed).

Please let me know if there is anything I can do.
